CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/BlobRequestHandler.java

```diff
@@ -37,6 +44,39 @@ public InputStream createImageContent(Args args) throws IOException {
         return RequestHandlerDispatcher.context.getAsset(imgFilePath[imgFilePath.length - 1]);
     }
 
+    public byte[] createImageContent(Args args) throws IOException {
+        String filePath = args.get("image");
+        if (filePath == null || filePath.isEmpty()) {
+            throw new IOException("Image content file path cannot be null");
+        }
+
+        String[] imgFilePath = filePath.split("/");
+        InputStream stream = RequestHandlerDispatcher.context.getAsset(imgFilePath[imgFilePath.length - 1]);
+        byte[] targetArray = new byte[stream.available()];
+        stream.read(targetArray);
+
+        return targetArray;
+    }
+
+    public URL createImageFileUrl(Args args) throws IOException, MalformedURLException {
+        String filePath = args.get("image");
+        if (filePath == null || filePath.isEmpty()) {
+            throw new IOException("Image content file path cannot be null");
+        }
+
+        String[] imgFilePath = filePath.split("/");
```

Comment:
If you are going to do this several times, it might make sense to make a little function to do it...

Reply:
good catch. the code was there because usually test cases requests were in unique form cross all platforms and all test cases. since this is a new test case and blob was not wildly used, I don't have to worry about regression. I am going to update this code, and going to update the python code also to ensure they are in sync. thanks!

Reply:
make change, the split path code has been removed.